README.md

```diff
@@ -1,21 +1,26 @@
-![](https://img.shields.io/badge/Microverse-blueviolet)
-
 # LeaderBoard
+
+
+
 In this activity I will set up a JavaScript project for the Leaderboard list app, using webpack and ES6 features, notably modules. I should develop a first working version of the app following a wireframe, but without styling - just focus on functionality. In following activities, I will consume the Leaderboard API using JavaScript async and await and add some styling.
+Built With
 
 # Built With
 
     html 5 , css
     visual studio code
     
-clone the project: https://github.com/Khanetmoi/leader.git
+# Getting started
 
-# 👤 Author1
+    
+   #    Install
+     https://khanetmoi.github.io/Portfolio-setup-and-mobile-version-skeleton/
```

Comment:
- [ ] I don't think this link belongs to this project.

- [ ] You need to outline detailed instructions on how to set up and run the app locally
  - git clone <repo link>
  - cd into cloned project
  - run npm install
  - run npm start